css/about.css

```diff
@@ -1,50 +1,115 @@
 /*
```

Comment:
Careful not to break your about.css styles. I would say either checkout `about.css` and `about.html` from `development` or merge `development` in and accept incoming changes.

Reply:
thank you! merged development into impact branch and have no conflicts